sda/cmd/auth/oidc.go

```diff
@@ -17,6 +17,7 @@ import (
 // OIDCIdentity represents an OIDC user instance
 type OIDCIdentity struct {
 	User                 string
+	Name                 string
```

Comment:
Based on how this struct is populated `Name` == `Profile`.
Why should we have the same value duplicated?

Reply:
I guess we should not... fixed.